cluster/gce/gci/configure-helper.sh

```diff
@@ -1562,6 +1650,51 @@ function start-etcd-servers {
   prepare-etcd-manifest "-events" "4002" "2381" "100m" "etcd-events.manifest"
 }
 
+# Replaces the variables in the network-proxy manifest file with the real values, and then
+# copy the file to the manifest dir
+# $1: value for variable "cpulimit"
+# $2: value for variable "server_port"
+# $3: value for variable "agent_port"
+# $4: value for bariable "admin_port"
+function prepare-network-proxy-manifest {
+  local -r temp_file="/tmp/network-proxy.yaml"
+  params=()
+  cp "${KUBE_HOME}/kube-manifests/kubernetes/gci-trusty/network-proxy.yaml" "${temp_file}"
+  sed -i -e "s@{{ *cpulimit *}}@$1@g" "${temp_file}"
+  params+=("--log-file=/var/log/network-proxy.log")
+  params+=("--logtostderr=false")
+  params+=("--log-file-max-size=0")
+  params+=("--server-ca-cert=${PROXY_SERVER_CA_CERT_PATH}")
+  params+=("--server-cert=${PROXY_SERVER_CERT_PATH}")
+  params+=("--server-key=${PROXY_SERVER_KEY_PATH}")
+  params+=("--cluster-ca-cert=${PROXY_AGENT_CA_CERT_PATH}")
+  params+=("--cluster-cert=${PROXY_AGENT_CERT_PATH}")
+  params+=("--cluster-key=${PROXY_AGENT_KEY_PATH}")
+  params+=("--mode=http-connect")
+  params+=("--server-port=$2")
+  params+=("--agent-port=$3")
+  params+=("--admin-port=$4")
+  proxy_args="$(convert-params-array-to-yaml "${params[@]}")"
+  echo "About to call sed -i -e \"s@{{ *proxy_args *}}@${proxy_args}@g\" \"${temp_file}\""
```

Comment:
Debug log? This should go to stderr if you intend to keep it.

Reply:
Fixed